target/linux/ramips/image/mt7621.mk

```diff
@@ -1068,6 +1068,20 @@ define Device/totolink_x5000r
 endef
 TARGET_DEVICES += totolink_x5000r
 
+define Device/tplink_archer-c6u-v1
+  $(Device/dsa-migration)
+  $(Device/tplink-safeloader)
+  IMAGE_SIZE := 15744k
```

Comment:
I'd prefer to move IMAGE_SIZE down like for re350-v1.

Reply:
I'm a bit confused how big the `IMAGE_SIZE` should be. I thought that number (15744k) should match the size of the `firmware (0x040000 - 0xf60000)` partition. I changed that to 6272k, which matches the size of the rootfs in a rootfs/rootfs-data split in a regular image.

Reply:
"move down", not "reduce" ...

Reply:
Sorry, I misunderstood. I moved it down as it's in re350.